virtdata-userlibs/src/main/java/io/nosqlbench/virtdata/userlibs/apps/VirtDataMainApp.java

```diff
@@ -46,7 +48,7 @@ public static void main(String[] args) {
     @Override
     public int applyAsInt(String[] args) {
         if (args.length == 0) {
-            System.out.println("Usage: app (" + APP_TESTMAPPER + "|" + APP_GENDOCS + "|" + APP_DIAGNOSE +")");
+            System.out.println("Usage: app (" + APP_TESTMAPPER + "|" + APP_BUILDDOCS + "|" + APP_GENDOCS +"|"+ APP_DIAGNOSE +")");
```

Comment:
We can remove the out-moded calls. Basically, *gendocs* should go away.

Reply:
Done, committing and pushing